### DISCUSSION TITLE
《老米的回归奖励》

### DISCUSSION THREAD

**Discussion by YuTinMIn** [share121/inter-knot#2543] [其他]
![双金](https://github.com/user-attachments/assets/7bfee977-88a2-441d-8fe2-4f0b70e5a98f)
我已经几个月没有玩原神了，都在玩绝区零和崩铁，然后我听说原神要送五星了，于是我云玩看一眼，发现还没有开始活动，然后我就用邮箱里的十连抽了一发，结果就双金了，可能是老米给的回归奖励吧，这个老米人真好啊（咸鱼音）
![咸鱼](https://github.com/user-attachments/assets/5f2a593a-e6f1-4b0f-b80c-2eab01d29809)

**Comment by hyperion-netizen** [comment]
不是哥們

**Comment by lllliufeng** [comment]
好好好

**Comment by Philia093-T** [comment]
不是，哥们你怎么现在回来呀？下个版本的回归玩家还会多送十连欸<img src="https://bbs-static.miyoushe.com/static/2023/08/23/675b943cf7767dada906620edd1e9b5d_7873468415922965365.png" alt="琳妮特-看戏" width="75" >

**Reply by YuTinMIn** [reply]
真的吗？

**Comment by 111snow-cherry** [comment]
哥们你回来早了

**Reply by YuTinMIn** [reply]
啊？
